### ISSUE TITLE
Proposal: scrollToOptions detection

### ISSUE THREAD

**Issue by asmarcz** [Modernizr/Modernizr#2433]
When using `scrollTo` function you can choose between two syntaxes. The scrollToOptions object is not supported in Edge, Safari and IE.

`scrollTo` scrolls to 0, 0 when it is supplied "invalid arguments" so the support can be detected by this.

I don't know whether this is something worth being in Modernizr. I would appreciate any help with my first contribution.

**Comment by rejas** [comment]
hi @asmarcz this could be indeed something for modernizr. what you need help with foremost? on how to write a PR in general or on how to write a test for modernizr?

**Comment by asmarcz** [comment]
@rejas Thank you for response. I will look into it in two days. I will ask here if I get stuck.

**Comment by asmarcz** [comment]
I have created test file in feature-detects folder and registered it in config-all.json. Then I generated bundle using `./bin/modernizr -c lib/config-all.json` however when I wanted to try out whether it works, I found out that there is an error in the bundle itself.

`modernizr.js:6866 Uncaught TypeError: Cannot read property 'removeChild' of null
    at Object.fn (modernizr.js:6866)
    at testRunner (modernizr.js:144)
    at modernizr.js:10169
    at modernizr.js:10188`

What should I do? Should I clone a different branch or something?
